plugins/in_tail/tail_file.c

```diff
@@ -370,8 +369,10 @@ static int process_content(struct flb_tail_file *file, size_t *bytes)
                                    out_sbuf->data,
                                    out_sbuf->size);
     }
-    else {
+    else if (file->skip_next) {
         *bytes = file->buf_len;
+    } else {
```

Comment:
Can you add newline after `}` ?

like 
```c
}
else {
```
https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#braces-usage-on-conditionals-loops-and-functions

Reply:
Done